### DISCUSSION TITLE
About the multi-classification

### DISCUSSION THREAD

**Discussion by XieXiao1997** [Project-MONAI/MONAI#3232] [Q&A]
When I tried to mimic the 3D classification in MONAI's example to build a multi-classification model,I had endless problems...like dimension error,target error..It's driving me crazy.
Who can give me some advise to do the 3D classify in my own dataset?
Thanks...

**Comment by Nic-Ma** [comment]
Please refer to: https://github.com/Project-MONAI/tutorials/blob/master/3d_classification/torch/densenet_training_dict.py.

Thanks.
